db/model/bot.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/bot.js
+ */
+const common = require('../helpers/common');
+const constants = require('../constants');
+
+const assoc = {};
+
+module.exports = function user(seq, dataTypes) {
+  const Bot = seq.define('Bot', {
+    name: {
+      type: dataTypes.STRING,
+      allowNull: false,
+      unique: true,
+      comment: 'Create a named bot',
+    },
+    location: {
+      type: dataTypes.STRING,
+      allowNull: false,
+      comment: 'The URL to load UI from',
```

Comment:
if it's always a url, sequelize has an out of the box validator to confirm that the string looks like a url